Makefile

```diff
@@ -13,7 +13,7 @@ all: lint maypanic statistics reachability dependencies static-commands render t
 install: taint_install backtrace_install cli_install
 
 lint: **/*.go
-	deadcode -test -filter ar-go-tools/analysis ./...
+	# deadcode -test -filter ar-go-tools/analysis ./...
```

Comment:
A comment explaining why this is disabled would be useful. Maybe we can enable it in the future if the tool gets fixed